skietbaan_fe/src/components/RegisterMemberStyles.css

```diff
@@ -32,15 +28,19 @@
     box-shadow: 2px 2px rgba(0,0,0,0.24);
   }
   @media screen and (min-width:768px)  {
-  .centre-register{
+  .centre-registerMember{
     width: 50%;
     left:25%;
-    top:1%;
+    top:8%;
   }
   }
-  
-  
-  
+  input.invalid{
+    border:1px solid red;
+  }
+  .invalid-message{
+    color:red;
+    font-size: 12px;
+  }
   .no-padding{
```

Comment:
I suggest you make a utilities css file for yourself, that manages your spacings around your elements. and you can even go as far as creating them for specific screen sizes.

I like to do something like this 

padding-h-15-xs{ padding:0 15px} which adds padding to the left and right of an element
and padding-v-15-xs{padding: 15px 0} which does the top and bottom

This having these classes will help you keep a consistent spacing and also help you tweak it for various screen sizes if you start adding those.

Reply:
or... use bootstrap which has all these classes built in 
Example: 
pl-1, pl-2, pl-3, pl-4... etc etc gives you various left paddings from small to large